File: Python/Fibonacci_Numbers/fib_recursion.py
# Function to generate Fibonacci sequence
def fibonacci(n):
    fib_sequence = [0, 1]
    while len(fib_sequence) < n:
        fib_sequence.append(fib_sequence[-1] + fib_sequence[-2])
    return fib_sequence[:n]

# Get number of iterations from the user
num_iterations = int(input("Enter the number of Fibonacci numbers to generate: "))

# Generate Fibonacci sequence
fib_sequence = fibonacci(num_iterations)

# Print the result
print(f"The first {num_iterations} Fibonacci numbers are: {fib_sequence}")

File: Java/Fibonacci_Numbers/fib_recursion.java
package Java.Fibonacci_Numbers;

import java.util.*;

public class fib_recursion {
    public static void main(String[] args){
        //Initalize variables
        int iterations;
        Scanner input = new Scanner(System.in);

        //Get number of iterations from user
        System.out.println("Enter number of iterations: ");
        iterations = input.nextInt();

        //Initalize array for Fibonacci sequence
        int[] sequence = new int[iterations];
        sequence[0] = 0;
        sequence[1] = 1;

        input.close();

        fibonacci(iterations, sequence);
    }

    public static void fibonacci(int n, int[] sequence){
        for(int i = 0; i < sequence.length; i++){
            if(i < n){

            }
        }
    }
}
